org.onionware.onionmedia.yml

```diff
@@ -0,0 +1,41 @@
+app-id: org.onionware.onionmedia
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: start.sh
+
+modules:     
+  - name: Desktop
+    buildsystem: simple
+    build-commands:
+      - cp -r . /app
+    sources:
+      - type: archive
+        url: https://github.com/onionware-github/OnionMedia-X/releases/download/2.0.0.0/flatpak-resources.zip
+        sha256: 4b97e11f562db8fbc2be0babed349f477af0ae848faaff5383cbc2459bceaf53
+        dest-filename: flatpak-resources.zip
+        
+  - name: OnionMedia
+    buildsystem: simple
+    build-commands:
+      - cp -r . /app/bin
+      - install -D start.sh /app/bin/start.sh
+      - chmod -R +x /app/bin
+    sources:
+      - type: archive
+        url: https://github.com/onionware-github/OnionMedia-X/releases/download/2.0.0.0/OnionMediaX_2.0.0.0-linux.X64.zip
```

Comment:
how will this work with the aarch64 flatpak that is built?

Reply:
Should be only available for x64 at the moment, I will add a flathub.json file. Can I close this PR and create a new one or can I edit this PR?

Reply:
why do you want to close this?

Reply:
How can I add files to a pull request? I added a flathub.json file to my fork, but it doesn't appear here.